UiPath.PowerShell.Tests/Cmdlets/MaintenanceModeTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using UiPath.PowerShell.Models;
+using UiPath.PowerShell.Tests.Util;
+using UiPath.PowerShell.Util;
+using UiPath.Web.Client201910.Models;
+using Cmdlet = System.Management.Automation.PowerShell;
+
+namespace UiPath.PowerShell.Tests.Cmdlets
+{
+    /// <summary>
+    /// Maintenance Mode PowerShell tests
+    /// </summary>
+    /// <remarks> In order to run these, MaintenanceMode should be enabled in Orchestrator (add key="MaintenanceMode.Enabled" value="true")</remarks>
```

Comment:
That was true while MM was in development. It should be enabled by default on-prem now.